user-guide/Cloud_Platform/CloudAdminApp/Controlling_remote_access.md

```diff
@@ -37,6 +37,20 @@ If you have the Owner or Admin role on dataminer.services for a DMS, you can ena
 
 1. Toggle the *Remote Access* settings to *On* or *Off*, depending on whether you want this to be enabled or not.
 
+   If the main *Remote Access* setting is enabled, the following settings are also available:
+
+   - *DataMiner Cube (desktop app)*: Enable or disable remote access to the DataMiner System(s) via the desktop app.
+
+   - *Remote Cube allowed IPs*: Only available if the *DataMiner Cube (desktop app)* setting is enabled. This setting allows you to restrict access to the DMS based on specified IP addresses.
```

Comment:
```suggestion
   - *Restict by IP address*: Only available if the *DataMiner Cube (desktop app)* setting is enabled. This setting allows you to restrict remote access to the DMS via the desktop app based on the specified client public IP addresses.
   > [!NOTE]
   > Only public IP addresses can be specified
```

Reply:
ideally add a tab to indicate this is a sub setting of "DataMiner Cube (desktop app)"